helix-rest/src/main/java/org/apache/helix/rest/server/resources/zookeeper/ZooKeeperAccessor.java

```diff
@@ -62,14 +62,20 @@ public Response get(@PathParam("path") String path, @QueryParam("command") Strin
     if (cmd == null) {
       return badRequest("Invalid ZooKeeper command: " + commandStr);
     }
+    if (path == null) {
```

Comment:
I don't think this PR is necessary because it is the caller's responsibility to correctly format the URL. With the correct URL, it has already been verified and tested that these endpoints work as expected.